sky/packages/sky/lib/src/fn3/framework.dart

```diff
@@ -721,6 +738,20 @@ class StatefulComponentElement extends BuildableElement<StatefulComponent> {
 class ParentDataElement extends StatelessComponentElement<ParentDataWidget> {
   ParentDataElement(ParentDataWidget widget) : super(widget);
 
+  void mount(Element parent, dynamic slot) {
```

Comment:
put a comment here saying what error this is trying to catch, so that if people get the assert they can more easily figure out what's wrong